siliconcompiler/tools/vpr/vpr.py

```diff
@@ -194,6 +194,25 @@ def runtime_options(chip, tool='vpr'):
     return options
 
 
+############################
+# Get common graphics files
+############################
+
+def get_common_graphics(chip, graphics_commands=[]):
```

Comment:
no need to pass in the graphics_commands list, you can just extend the list on the return (also this doens't work correctly)

Reply:
Fixed.